html_css/semantic_html.md

```diff
@@ -0,0 +1,94 @@
+* When you use an input element, you should always create a relationship between it and a `<label>` element. A `<label>` provides context for what an input actually means to assistive technologies, announcing the label contents each time the input is announced. Not only that, but a proper `<label>` increases the clickable area of the input itself, which is useful for users who have trouble clicking on smaller items. There are two ways you can create this relationship:
+
+~~~html
+<!-- Useful when you need the input itself to have an ID -->
+<label for='name-input'>Name</label>
+<input type="text" id="name" />
+
+<!-- Look, Ma, no ID! -->
+<label>
+  Name
+  <input type="text" />
+</label>
+~~~
+
+* Continuing with inputs, you should always use the `type` that makes the most sense for its intended use. `type="text"`makes more sense for a name or address field, while `type="email"` or `type="tel"` would of course make more sense for an e-mail or telephone field, respectively. For certain devices, using the correct `type` may show only the required or additional characters on the keyboard. A `type="tel"` input, for example, might make it much easier for users to fill out their phone number by providing larger, numerical-only keys on mobile or tablet devices.
+* When you want to present a list of some sort to a user, you should use the appropriate list element (`ol`, `<ul>`, or `<dl>`) and their related list item elements. This will not only let the user know when they are entering or exiting a list, but also how many items are in the list.
+
+### Headings and Landmarks
+
+Headings are the `<h1>` through `<h6>` elements, and like the name implies, these elements act as headings to sections of a page. Landmarks, on the other hand, are HTML elements that act as regions of a page. There are seven native HTML elements that define these landmark regions:
+
+* `<aside>`
+* `<footer>`
+* `<form>`
+* `<header>`
+* `<main>`
+* `<nav>`
+* `<section>`
+
+By properly using landmarks and headings, you provide users of assistive technologies a more operable and understandbale page: not only can screen readers navigate a page via landmarks and headings by opening a menu in their screen reader, but these elements also have their roles announced to provide additional context.
+
+If you were to use only `<div>` elements to act as these landmarks and headings, maybe adding in some CSS to visually style them, then a screen reader user would have to go through the entire page just to get to a specific section, and they may not be able to actually tell what is a heading or a landmark on the page. To see how a screen reader interacts with these elements, watch [Why headings and landmarks are so important](https://www.youtube.com/watch?v=vAAzdi1xuUY&list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g&index=19). This video also quickly mentions some things you *shouldn't* do with the heading elements.
+
```

Comment:
lets add an assignment section here too.. just pull the links from above down here.